### PR TITLE
FEATURE: Require spec helpers for plugins

### PR DESCRIPTION
**Pull request by rimian**
This will allow plugin developers to load fixtures and helpers in specs.  

  * Follows any symlinked plugins

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -29,6 +29,14 @@
   Dir[Rails.root.join("spec/support/**/*.rb")].each {|f| require f}
   Dir[Rails.root.join("spec/fabricators/*.rb")].each {|f| require f}
 
+  # Require plugin helpers at plugin/[plugin]/spec/plugin_helper.rb (includes symlinked plugins).
+  if ENV['LOAD_PLUGINS'] == "1"
+    Dir[Rails.root.join("plugins/**")].each do |plugin|
```
Comment (SamSaffron) [inline spec/rails_helper.rb]: this is fine, but I think "plugins/*" would suffice here

Comment (SamSaffron) [inline spec/rails_helper.rb]: actually ... why not search for them properly and avoid the double lookup

```
Dir[Rails.root.join("plugins/*/spec/plugin_helper.rb")]
```

**Comment by rimian** [comment]
I tried  that before but globs don't work with symlinks. :(

File: spec/rails_helper.rb (hunk 1 above)
Comment (rimian) [inline spec/rails_helper.rb]: Hey that worked! Thanks.

**Comment by SamSaffron** [comment]
cool, seems fine to me them 👍

**Pull request merged**